.ibm/pipelines/jobs/operator.sh

```diff
@@ -8,9 +8,15 @@ install_rhdh_operator() {
 
   # Make sure script is up to date
   rm -f /tmp/install-rhdh-catalog-source.sh
-  curl -L https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/main/.rhdh/scripts/install-rhdh-catalog-source.sh > /tmp/install-rhdh-catalog-source.sh
+  curl -L "https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/${RELEASE_BRANCH_NAME}/.rhdh/scripts/install-rhdh-catalog-source.sh" > /tmp/install-rhdh-catalog-source.sh
   chmod +x /tmp/install-rhdh-catalog-source.sh
-  bash -x /tmp/install-rhdh-catalog-source.sh --next --install-operator rhdh
+  if [ "$TAG_NAME" == "next" ]; then
```

Comment:
@zdrapela for PR specific runs it will give false positive, as tag name would not be next on main or other branchs.

Reply:
Hmm, then we probably want to compare with the target branch `PULL_BASE_REF` (source: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables) 
```suggestion
  if [ "$PULL_BASE_REF" == "main" ]; then
```
Correct?